tajo-core/pom.xml

```diff
@@ -690,6 +696,37 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>parallel-test</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
Thank you for reporting a issue. I did not test on MacOS, and found that maven on MacOS searches the invalid classname like 'rg.apache...'. I will keep finding a workaround on this issue.